bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/events/ItemEventFactory.java

```diff
@@ -93,7 +96,7 @@ private Event createCommandEvent(String topic, String payload, String source) {
         ItemEventPayloadBean bean = deserializePayload(payload, ItemEventPayloadBean.class);
         Command command = null;
         try {
-            command = (Command) parse(bean.getType(), bean.getValue());
+            command = (Command) parse(bean.getType() + TYPE_POSTFIX, bean.getValue());
```

Comment:
> open/closed principle

We could use a `parseType` and `parseSimpleClassName` method then no one needs to be changes anymore. :wink: